operations/node_management/Adding_a_Liquid-cooled_blade_to_a_System_Using_SAT.md

```diff
@@ -47,6 +47,43 @@ This procedure will add a liquid-cooled blade to an HPE Cray EX system.
    sat swap blade --action enable <SLOT_XNAME>
    ```
 
+1. Clear out the existing Redfish event subscriptions from the BMCs on the blade.
+
+    1. (`ncn#`) Set the environment variable `SLOT` corresponding to the blades location:
+
+        ```bash
+        SLOT=<SLOT_XNAME>
+        ```
+
+    1. (`ncn#`) Clear the Redfish event subscriptions:
+
+        ```bash
+        for BMC in $(cray hsm inventory  redfishEndpoints list --type NodeBMC --format json | jq .RedfishEndpoints[].ID -r | grep $SLOT); do
+            PASSWD=$(cray scsd bmc creds list --targets $BMC --format json | jq .Targets[].Password -r)
+            SUBS=$(curl -sk -u root:$PASSWD https://${BMC}/redfish/v1/EventService/Subscriptions | jq -r '.Members[]."@odata.id"')
+            for SUB in $SUBS; do
+                echo "Deleting event subscription: https://${BMC}${SUB}" 
+                curl -i -sk -u root:$PASSWD -X DELETE https://${BMC}${SUB}
```

Comment:
Would be better to avoid putting the password on the curl command line, because that is visible in the output of `ps -ef` for any users logged into the system, regardless of access level.